account_analytic_wip/models/account_analytic.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+
+
+class AccountAnalytic(models.Model):
+    """
+    An Analytic Account can have one or more related Tracking Items.
+    Depending on the extensions installed, you might have
+    Tracking Items per Task or per Manufacturing Order.
+    """
+
+    _inherit = "account.analytic.account"
+
+    analytic_tracking_item_ids = fields.One2many(
+        "account.analytic.tracking.item", "analytic_id", string="Tracking Items"
+    )
+
+    @api.model
+    def create(self, vals):
+        """
+        A default Tracking Item is automatically created.
+        It will collect Atual Amounts not linked to a specific Tracking Item.
+        """
+        new = super().create(vals)
+        self.env["account.analytic.tracking.item"].create({"analytic_id": new.id})
```

Comment:
There seems to be an error in this method and it might be because it's missing a return new line which seems to fix the issue. Error happens anywhere where an analytic account is trying to be created.

Error: 
Odoo Server Error
Traceback (most recent call last):
  File "/opt/odoo/odoo-boti/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/odoo-boti/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 347, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 912, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 531, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/odoo-boti/odoo/addons/web/controllers/main.py", line 1389, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/odoo-boti/odoo/addons/web/controllers/main.py", line 1381, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/odoo-boti/odoo/odoo/api.py", line 394, in call_kw
    result = _call_kw_model_create(method, model, args, kwargs)
  File "/opt/odoo/odoo-boti/odoo/odoo/api.py", line 375, in _call_kw_model_create
    return result.id if isinstance(args[0], Mapping) else result.ids
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 639, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/odoo-boti/odoo/odoo/http.py", line 315, in _handle_exception
    raise exception.with_traceback(None) from new_cause
AttributeError: 'NoneType' object has no attribute 'id'